### PR TITLE
Create 2017-06-22.yaml

### PR DESCRIPTION
**Pull request by sseidelmann**

### CODE REVIEW

File: shopware/shopware/2017-06-22.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+title:     Remote Code Execution Vulnerability
+link:      http://community.shopware.com/Security-Update-06-2017_detail_2014_482.html
```
Comment (stloyd) [inline shopware/shopware/2017-06-22.yaml]: Link should be to english version: http://en.community.shopware.com/_detail_2015.html

File: shopware/shopware/2017-06-22.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+
+title:     Remote Code Execution Vulnerability
+link:      http://en.community.shopware.com/_detail_2015.html
+branches:
+    4.2:
+        time:     2017-06-22 10:07:00
+        versions: ['>=4.2.0', '<4.3.0']
+    4.3:
+        time:     2017-06-22 10:07:00
+        versions: ['>=4.3.0', '<4.4.0']
```
Comment (stof) [inline shopware/shopware/2017-06-22.yaml]: I would even say `<5.0.0`, as there is not non-vulnerable 4.x version AFAICT

File: shopware/shopware/2017-06-22.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (fabpot) [inline shopware/shopware/2017-06-22.yaml]: extra line could be removed

**Comment by stof** [comment]
@sseidelmann can you make the fixes ?

**Comment by fabpot** [comment]
@sseidelmann any news?

**Comment by fabpot** [comment]
@marcomenzel Sure, if you can take over this PR, that would be great. Thanks.

**Comment by fabpot** [comment]
Closing in favor of #234

**Pull request closed without merging**